README.md

```diff
@@ -225,6 +225,22 @@ Alternatively you can use the `--message-template-drain` and `--message-template
 --message-template-drain="Draining node %s part of *my-cluster* in region *xyz*"
 ```
 
+### Notifications 
+
+When you specify a formatted URL using `--notify-url`, kured will notify
+about draining and rebooting nodes across a list of technologies. Here is 
+the syntax:
+
+slack:           `slack://tokenA/tokenB/tokenC`
+
+rocketchat:      `rocketchat://[username@]rocketchat-host/token[/channel|@recipient]`
+
+teams:           `teams://token-a/token-b/token-c`
```

Comment:
have all of those been tested and working?

Reply:
RC was tested. No idea about teams. You can check the performance by the number of the issues there.

Reply:
If it's not tested, I would remove it. People can extend the documentation when we have tested and ensure working.